### ISSUE TITLE
duplicate symbol in arm64

### ISSUE THREAD

**Issue by Whirlwind** [nishanths/YWeatherAPI#1]
There are too many duplicate symbol when build in arm64.

```
duplicate symbol _kYWAIndex in:
    /Users/whirlwind/Library/Developer/Xcode/DerivedData/SmartLivingRoom-dipnzxjwucgdgcfqcxkfqbnnktsb/Build/Intermediates/SmartLivingRoom.build/Debug-iphoneos/SmartLivingRoom.build/Objects-normal/arm64/SLRClockViewController.o
    /Users/whirlwind/Library/Developer/Xcode/DerivedData/SmartLivingRoom-dipnzxjwucgdgcfqcxkfqbnnktsb/Build/Products/Debug-iphoneos/libPods-SmartLivingRoom-YWeatherAPI.a(YWeatherAPI.o)
duplicate symbol _kYWAEmptyValue in:
    /Users/whirlwind/Library/Developer/Xcode/DerivedData/SmartLivingRoom-dipnzxjwucgdgcfqcxkfqbnnktsb/Build/Intermediates/SmartLivingRoom.build/Debug-iphoneos/SmartLivingRoom.build/Objects-normal/arm64/SLRClockViewController.o
    /Users/whirlwind/Library/Developer/Xcode/DerivedData/SmartLivingRoom-dipnzxjwucgdgcfqcxkfqbnnktsb/Build/Products/Debug-iphoneos/libPods-SmartLivingRoom-YWeatherAPI.a(YWeatherAPI.o)
duplicate symbol _kYWAFiveDayForecasts in:
    /Users/whirlwind/Library/Developer/Xcode/DerivedData/SmartLivingRoom-dipnzxjwucgdgcfqcxkfqbnnktsb/Build/Intermediates/SmartLivingRoom.build/Debug-iphoneos/SmartLivingRoom.build/Objects-normal/arm64/SLRClockViewController.o
    /Users/whirlwind/Library/Developer/Xcode/DerivedData/SmartLivingRoom-dipnzxjwucgdgcfqcxkfqbnnktsb/Build/Products/Debug-iphoneos/libPods-SmartLivingRoom-YWeatherAPI.a(YWeatherAPI.o)
duplicate symbol _kYWAConditionNumber in:
    /Users/whirlwind/Library/Developer/Xcode/DerivedData/SmartLivingRoom-dipnzxjwucgdgcfqcxkfqbnnktsb/Build/Intermediates/SmartLivingRoom.build/Debug-iphoneos/SmartLivingRoom.build/Objects-normal/arm64/SLRClockViewController.o
    /Users/whirlwind/Library/Developer/Xcode/DerivedData/SmartLivingRoom-dipnzxjwucgdgcfqcxkfqbnnktsb/Build/Products/Debug-iphoneos/libPods-SmartLivingRoom-YWeatherAPI.a(YWeatherAPI.o)
duplicate symbol _kYWACondition in:
    /Users/whirlwind/Library/Developer/Xcode/DerivedData/SmartLivingRoom-dipnzxjwucgdgcfqcxkfqbnnktsb/Build/Intermediates/SmartLivingRoom.build/Debug-iphoneos/SmartLivingRoom.build/Objects-normal/arm64/SLRClockViewController.o
    /Users/whirlwind/Library/Developer/Xcode/DerivedData/SmartLivingRoom-dipnzxjwucgdgcfqcxkfqbnnktsb/Build/Products/Debug-iphoneos/libPods-SmartLivingRoom-YWeatherAPI.a(YWeatherAPI.o)
duplicate symbol _kYWALongDescription in:
    /Users/whirlwind/Library/Developer/Xcode/DerivedData/SmartLivingRoom-dipnzxjwucgdgcfqcxkfqbnnktsb/Build/Intermediates/SmartLivingRoom.build/Debug-iphoneos/SmartLivingRoom.build/Objects-normal/arm64/SLRClockViewController.o
    /Users/whirlwind/Library/Developer/Xcode/DerivedData/SmartLivingRoom-dipnzxjwucgdgcfqcxkfqbnnktsb/Build/Products/Debug-iphoneos/libPods-SmartLivingRoom-YWeatherAPI.a(YWeatherAPI.o)
duplicate symbol _kYWAShortDescription in:
    /Users/whirlwind/Library/Developer/Xcode/DerivedData/SmartLivingRoom-dipnzxjwucgdgcfqcxkfqbnnktsb/Build/Intermediates/SmartLivingRoom.build/Debug-iphoneos/SmartLivingRoom.build/Objects-normal/arm64/SLRClockViewController.o
    /Users/whirlwind/Library/Developer/Xcode/DerivedData/SmartLivingRoom-dipnzxjwucgdgcfqcxkfqbnnktsb/Build/Products/Debug-iphoneos/libPods-SmartLivingRoom-YWeatherAPI.a(YWeatherAPI.o)
...
```

**Comment by Whirlwind** [comment]
see http://stackoverflow.com/questions/10957152/duplicate-symbol-error-global-constant

**Comment by nishanths** [comment]
@Whirlwind Thanks for reporting. I followed the accepted Stack Overflow answer. The issue should be fixed in v1.0.5 via 2c6f09a18c3795a1f823625839512eb353ab460f.
